core/gui/custom-webpack.config.js

```diff
@@ -7,5 +7,10 @@ module.exports = {
         include: [require("path").resolve(__dirname, "node_modules/monaco-editor")],
       },
     ],
+    parser: {
+      javascript: {
+          url: true
+      }
+    }
```

Comment:
@mengw15 what's the purpose of this change? I found removing it has no effect.

Reply:
I was facing an issue with loading vscode.theme-default I remember. And I checked Monaco's example:

"
// this is required for loading .wasm (and other) files. For context, see https://stackoverflow.com/a/75252098 and https://github.com/angular/angular-cli/issues/24617
        parser: {
            javascript: {
                url: true
            }
        }
"